src/exoscale/telex/interceptor/ring1.clj

```diff
@@ -57,9 +58,11 @@
 
 (def form-params-interceptor
   {:name ::form-params
-   :enter (-> interceptor/encode-query-params
-              (ix/in [:ring1.request/form-params])
-              (ix/out [:ring1.request/body]))})
+   :enter (fn [{:as ctx :ring1.request/keys [form-params]}]
+            (cond-> ctx
+              (seq form-params)
+              (assoc :ring1.request/body
+                     (interceptor/encode-query-params form-params))))})
```

Comment:
Why not using `ix/when` ?
```suggestion
  :enter (-> interceptor/encode-query-params
              (ix/in [:ring1.request/form-params])
              (ix/out [:ring1.request/body])
              (ix/when :ring1.request/form-params))})
```

Reply:
very good point, forgot about it :) you know interceptor better than me now!

Reply:
I ll also add a check on presence of body, if by mistake the user mixes both, body takes over.